frontend/packages/dev-console/src/components/gitops/GitOpsServiceDetailsSection.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { Link } from 'react-router-dom';
+import {
+  StackItem,
+  Card,
+  CardTitle,
+  CardBody,
+  Label,
+  Split,
+  SplitItem,
+} from '@patternfly/react-core';
+import { ResourceLink } from '@console/internal/components/utils';
+import './GitOpsServiceDetailsSection.scss';
+
+interface GitOpsServiceDetailsSectionProps {
+  services: any;
+}
+
+const GitOpsServiceDetailsSection: React.FC<GitOpsServiceDetailsSectionProps> = ({ services }) => {
+  return (
+    <>
+      {_.map(services, (service) => (
```

Comment:
no need to use fragment you can directly return _.map

Reply:
It doesnt seem like we can directly return map